src/shipit_static_analysis/shipit_static_analysis/__init__.py

```diff
@@ -41,6 +41,12 @@ def as_markdown(self):
         '''
         raise NotImplementedError
 
+    def as_diff(self):
```

Comment:
Nit: you could make this an abstract method (https://docs.python.org/3/library/abc.html#abc.abstractmethod)

Reply:
:100: